test/fit_tests/config/global_config.json

```diff
@@ -75,7 +75,12 @@
       },
     "skupack": [
       "https://github.com/RackHD/on-skupack"
-      ]
+      ],
+    "skupacks":{"Quanta T41":"http://web.hwimo.lab.emc.com/autotest/skupack/quanta-t41/quanta-t41_t41.tar.gz",
```

Comment:
Let's move this into a separate config file - fw_config.json.  You are referencing an internal server for these tests, so it relies on running in a specific test environment.